torsiondrive/qm_engine.py

```diff
@@ -12,12 +13,42 @@ def check_all_float(iterable):
     except ValueError:
         return False
 
+def make_constraints_dict(constraints_string):
+    """ Create an ordered dictionary with constraints specification. """  
+    constraints_mode = None
+    constraints_dict = OrderedDict([('freeze', []),('set', [])])
```

Comment:
I guess as we are working with Python 3 now `OrderedDict` can be replaced by `dict`

Reply:
Is torsiondrive Python 2.7 compatible? If so, I would prefer to use OrderedDict. If not, then we can replace with dict.

Reply:
I think I didn't make additional efforts to ensure Python 2.7 compatibility :)